src/components/study-pane.js

```diff
@@ -689,6 +710,7 @@ const StudyPane = (props) => {
     useEffect(() => {
         websocketExpectedCloseRef.current = false;
         dispatch(openStudy(studyUuid));
+        checkStudyExists();
         loadNetwork();
```

Comment:
Is it interesting to call loadNetwork, loadGeoData, loadTree if study doesn't exists ?
I know we would like to speed up loading with parallelization but, maybe checking Study existence is a prerequisite.